app/views/layouts/application.html.erb

```diff
@@ -17,8 +18,12 @@
         <div class="menu-item">
           <%= menu_link_to 'All users', users_path %>
         </div>
+        <div class="menu-item">
+          <%= menu_link_to 'Friends', friends_path %>
+        </div>
       </div>
       <% if current_user %>
```

Comment:
- [ ] Again, avoid logic in views.

Reply:
This isn't my code, it is part of the template and it's wrong for you to request me to work on it.